app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
```

Comment:
- [ ] OPTIONAL
Foreign key already declared in the _User_ model, and hence no need to declare it again in a referenced model. _Sorry, this **important step** was missed in the previous review_.

Simply do something like:
```ruby
# app/models/comment.rb
belongs_to :author, class_name: 'User'